<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de eventos ‚Äî Participantes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="header" native="true">
      <layout class="QHBoxLayout" name="headerLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTitulo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>LISTA DE PARTICIPANTES: [Nombre del Evento]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSubirCSV">
         <property name="text">
          <string>Subir CSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenerarMesas">
         <property name="text">
          <string>Generar mesas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAnadir">
         <property name="text">
          <string>+ A√±adir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditar">
         <property name="text">
          <string>Editar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminar">
         <property name="text">
          <string>Eliminar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="wrapTabla">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="wrapTablaLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QWidget" name="barraSeccion" native="true">
         <layout class="QHBoxLayout" name="barraSeccionLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblSeccion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Invitados &gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnMenuTabla">
            <property name="toolTip">
             <string>Men√∫ de tabla</string>
            </property>
            <property name="text">
             <string>‚ñæ</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblInvitados">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>125</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>125</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>23</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>125</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Apellido</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Pref de estar</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prefetencias de con quien no estar</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="buscarWrap" native="true">
      <layout class="QHBoxLayout" name="buscarLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="txtBuscar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Buscar por nombre de invitado</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnBuscar">
         <property name="toolTip">
          <string>Buscar</string>
         </property>
         <property name="text">
          <string>üîç</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
